cypress/e2e/Admin/results/TableByQuestion.cy.js

```diff
@@ -0,0 +1,203 @@
+import {
+  APP_SETTING_NAMES,
+  PERMISSION_LEVELS,
+} from '../../../../src/config/constants';
+import { CONTEXTS } from '../../../../src/config/contexts';
+import {
+  NAVIGATION_RESULT_BUTTON_CY,
+  buildTableByQuestionAnswerHeader,
+  buildTableByQuestionCorrectHeader,
+  buildTableByQuestionCy,
+  buildTableByQuestionDateHeader,
+  buildTableByQuestionTableBodyCy,
+  buildTableByQuestionUserHeader,
+  dataCyWrapper,
+} from '../../../../src/config/selectors';
+import { APP_DATA } from '../../../fixtures/appData';
+import { APP_SETTINGS2 } from '../../../fixtures/appSettings';
+
+describe('Table by Question', () => {
+  /**
+   * Test the table by question view for a few question and some user answers
+   */
+  it('Table by Question correctly display data', () => {
```

Comment:
What about empty app data?

Reply:
Yes, you're right, I added a test to check the behavior when there isn't answered question yet.